src/databricks/labs/ucx/runtime.py

```diff
@@ -23,6 +23,127 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_view_definition(cfg: WorkspaceConfig) -> str:
+    return f"""vw_failure_summary AS WITH failuretab (failures, component) AS (
+        SELECT
+          failures,
+          "jobs" AS component
+        FROM
+          hive_metastore.{cfg.inventory_database}.jobs
+        UNION ALL
+        SELECT
+          failures,
+          "clusters" AS component
+        FROM
+          hive_metastore.{cfg.inventory_database}.clusters
+        UNION ALL
+        SELECT
+          failures,
+          "global init scripts" AS component
+        FROM
+          hive_metastore.{cfg.inventory_database}.global_init_scripts
+        UNION ALL
+        SELECT
+          failures,
+          "pipelines" AS component
+        FROM
+          hive_metastore.{cfg.inventory_database}.pipelines
+      )
+    SELECT
+      issue,
+      component,
+      COUNT(*) AS issue_count,
+      IF (
+        component = 'jobs',
+        round(
+          (issue_count / (
+            SELECT
+              count(*)
+            FROM
+              hive_metastore.{cfg.inventory_database}.jobs
+        ) * 100) , 2),
+        'NA'
+      ) AS jobs_issue_percentage,
```

Comment:
Not sure if we need these percentage columns here, especially when we add tables